testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -61,6 +64,11 @@ class MiniAppDisplayActivity : BaseActivity() {
         if (intent.hasExtra(miniAppTag) || intent.hasExtra(appIdTag)) {
             appId = intent.getStringExtra(appIdTag) ?: ""
 
+            miniAppId = if (intent.hasExtra(miniAppTag))
```

Comment:
I know we have `appId` before. Let's refactor to have only `appId` or `miniAppId`.

Reply:
Ok, I'll use separate PR for refactoring this because it will require to check other pages as well.

Reply:
ah please help me to do it in this PR too 🙏 
it's not much so we don't need to wait for another PR.

Reply:
Ok, Thanks!